external/ChebTools/ChebTools.cpp

```diff
@@ -0,0 +1,882 @@
+#include "ChebTools/ChebTools.h"
```

Comment:
Is this file taken from here https://github.com/usnistgov/ChebTools/blob/master/src/ChebTools.cpp ?
I think it is a good idea to acknowledge the authors. Either by adding some comments in the beginning of the file or even better by adding a short README in `external/ChebTools` containing (at least) the original path of the library, the version you used and the licence.